main.go

```diff
@@ -136,19 +137,19 @@ func loadEnvironmentVariables() {
 	prefix := "BACKUP_"
 	// Load environment variables
 	err := koanfInstance.Load(env.Provider(prefix, ".", func(s string) string {
-		s = strings.TrimLeft(s, prefix)
+		s = strings.TrimPrefix(s, prefix)
 		s = strings.Replace(strings.ToLower(s), "_", "-", -1)
 		return s
 	}), nil)
 	if err != nil {
-		setupLog.Error(err, "could not load environment variables")
+		_, _ = fmt.Fprintf(os.Stderr, "could not load environment variables: %v\n", err)
```

Comment:
Can you explain why you changed the logging here? So you can use the format strings?

Also maybe put it into its own commit.

Reply:
It's in the PR:
> Also prints config loading errors to stderr in loadEnvironmentVariables, since the setupLog isn't initiated at this time.

Reply:
Right, reading the whole thing would help 🙈